demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -118,12 +118,13 @@ def collect_all_inputs_in_use(self) -> Set[str]:
         result: set = set()
         with open(self.file_path) as f:
             playbook_text = f.read()
-        all_inputs_occurrences = re.findall(r"inputs\.[-\w ]+", playbook_text)
+        # all_inputs_occurrences = re.findall(r"inputs\.[-\w ]+", playbook_text)
+        all_inputs_occurrences = re.findall(r"inputs\.(.*)", playbook_text)
         for input in all_inputs_occurrences:
             input = input.strip()
-            splitted = input.split(".")
-            if len(splitted) > 1 and splitted[1] and not splitted[1].startswith(" "):
-                result.add(splitted[1])
+
+            if input and not input.startswith(" ") and input != '"':
```

Comment:
@adi88d can you please document this part so it's easier to understand what we're checking?